inference-engine/ie_bridges/c/include/c_api/ie_c_api.h

```diff
@@ -371,6 +371,18 @@ INFERENCE_ENGINE_C_API(void) ie_core_versions_free(ie_core_versions_t *vers);
  */
 INFERENCE_ENGINE_C_API(IE_NODISCARD IEStatusCode) ie_core_read_network(ie_core_t *core, const char *xml, const char *weights_file, ie_network_t **network);
 
+/**
+ * @brief Reads the model from an xml string and a blob of the bin part of the IR. Use the ie_network_free() method to free memory.
+ * @ingroup Core
+ * @param core A pointer to ie_core_t instance.
+ * @param xml_content Xml content of the IR.
```

Comment:
also need to add description for `xml_content_size`

Reply:
looks fine to me, but doesnt it make sence to change pointer to const void * instead of const char* ?

Reply:
I used a const char * because the C++ API takes a std::string.

Reply:
const char* not used in describing memory pointers(strdup vs memcpy), more over it is easie to mix with null terminated char arrays,  Also using dedicated types like intptr_t for pointes not good as well since it in optional even in C99 standard. 
More over if you are dealing with locale sentive strings or encrypted mem,  const char might not be used in caller code at all - application simply will use different storage type, and casting it to char* won't make it an ascii chars. 
So either generic const void*, or designated const uint8_t* to easier calculate offsets, etc using pointers math.